src/components/LoadingSpinner.tsx

```
@@ -0,0 +1,15 @@
+import clsx from 'clsx'
+
+export default function LoadingSpinner({ message }: { message?: string }) {
+  return (
+    <div className="flex-col-center h-full gap-6" role="status">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)